src/scenes/mainmenu.lua

```diff
@@ -287,6 +287,170 @@ Press the manage button below.]])
     end
 end
 
+function scene.buttonWithIcon(icon, text, green, cb)
+    return uie[green and "buttonGreen" or "button"](
+        uie.row({ uie.icon(icon):with({ scale = 21 / 256 }), uie.label(text) }):with({
+            clip = false,
+            cacheable = false
+        }):with(uiu.styleDeep), function()
+            cb()
+        end
+    ):with(uiu.fillWidth)
+end
+
+function scene.openLoennMenu()
+    -- open an alert, with a "loading" message at first
+    local alertMessage = alert({
+        title = "Lönn (Map Editor)",
+        body = uie.column({
+            uie.row({
+                uie.spinner():with({
+                    width = 16,
+                    height = 16
+                }),
+                uie.label("Loading")
+            }):as("loading")
+        }):with(uiu.fillWidth),
+        buttons = {
+            { "Close", function(container)
+                container:close()
+            end }
+        },
+        init = function(container)
+            container:findChild("box"):with({
+                width = 600, height = 400
+            })
+            container:findChild("buttons"):with(uiu.bottombound)
+        end
+    })
+
+    sharp.getLoennLatestVersion():calls(function (t, data)
+        local latestVersion = data.Item1
+        local downloadLink = data.Item2
+
+        -- version info
+        local content = {
+            uie.label(
+                (config.loennInstalledVersion == ""
+                    and "Lönn is currently not installed."
+                    or "Currently installed version: " .. config.loennInstalledVersion) .. "\n" ..
+                "Latest version: " .. latestVersion .. "\n" ..
+                "Install folder: " .. config.loennRootPath
```

Comment:
Could be made a bit more readable with a `string.format` or `table.concat` on lines? Works just fine as it is though.